release-pdf.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ $# -ne 2 ]; then
+    echo 'usage: ./release-pdf.sh <release version> <pdf name>'
+    exit 1
+fi
+bundle install && PO_TOKEN=cea9a05c311ed1996b973477f01773f847047f37 bundle exec ruby release-notes.rb $1 > $2.md
```

Comment:
😮 this token wasn't meant to be disclosed. GitHub has invalidated it automatically, so no dramas.

For context, the `PO_TOKEN` env var was defined exactly to avoid having to store the token in the repo, by letting the consumer setting it at call time.

I think if you removed the `PO_TOKEN=...` from here and call this script like `PO_TOKEN=... ./release-pdf.sh 1.2.3 pdf_name` it will still work.

Reply:
Worth renaming the env var to something like `PO_SECRET_TOKEN`?